spec/support/jasmine.json

```diff
@@ -0,0 +1,3 @@
+{
+  "stopSpecOnExpectationFailure": true
```

Comment:
Although this sounds _awesome_, Atom runs Jasmine v1.3.0, which [doesn't support](http://stackoverflow.com/questions/22119193/stop-jasmine-test-after-first-expect-fails) this unfortunately.